### DISCUSSION TITLE
Question about Train, Test loss, and Test Metric

### DISCUSSION THREAD

**Discussion by dapyrate** [lululxvi/deepxde#711] [Q&A]
Hi,
I ran [Poisson on L-shaped domain code](https://github.com/shawnrosofsky/HAL-Physics-Informed-AI-Tutorial/blob/main/Poisson_Lshape.ipynb) using DeepXDE.
As you can see for step 0, 1000, 2000, and ... I get a list of loss Train and Test loss numbers of length `2`.
For example for step `0` I get `[1.26e+00, 2.07e-01]` for train loss and `[1.26+00, 2.07e-01]` for test loss. Should I not get a scalar instead of a list of numbers for train and test loss? What is the first and second component in the list representing exactly? Are they representing boundary conditions loss? (How can I find out which component represents which boundary condition?)
In addition, why is test metric `[]` (an empty list)? What is test metric exactly representing? I also could not find any explanation in the documentation.

Thanks in advance

![ModelTraining](https://user-images.githubusercontent.com/61572669/170857090-3814b6f7-47c0-4d2e-9d2d-443a8d74802e.png)

**Comment by lululxvi** [comment]
See FAQ.
